src/react/client/client.spec.js

```diff
@@ -0,0 +1,52 @@
+import { ipcRenderer } from 'electron'
+import { invokeChannel } from './client'
+import { invokeAlertDialog } from './alertDialog'
+
+jest.mock('electron', () => ({ ipcRenderer: { invoke: jest.fn() } }), { virtual: true })
+jest.mock('./alertDialog', () => ({ invokeAlertDialog: jest.fn() }))
+
+const logInfoMock = jest.fn()
+const logErrorMock = jest.fn()
+window.frontendLogger = { info: logInfoMock, error: logErrorMock }
```

Comment:
This direct mocking might cause problems elsewhere. You might need to restore original behaviour or use some jest mocking / spy.

Reply:
The restoring is done in `afterEach` block in [setupTests.js](src/setupTests.js)

Reply:
It is done in global `afterEach` block, see: https://github.com/DioryMe/diory-browser-electron/pull/79/files#diff-f1995b7576ce098f06844c16df12561f4694c56c84869ec5180933e652f56862

Although I'm not sure what it is exactly doing but that's the solution I ended up to...